README.md

```diff
@@ -1 +1,185 @@
-# hello-microverse
\ No newline at end of file
+<a name="readme-top"></a>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Microverse Hello World Project] <a name="about-project"></a>
+
+**[Microverse Hello World Project]** is a project whose goal is to master all of the tools and best practices of web development.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+  </ul>
+</details>
+
+- HTML, CSS
+- No Frameworks
+- Technologies Used- HTML, CSS, GIT, Linters (GitHub Actions, Lighthouse, Webhint, Stylelint, Eslint)
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[A webpage that displays 'Hello Microverse!']**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Add a link to your deployed project.
+
+- [Live Demo Link](https://seyioyemade.github.io/hello-world/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+You need to have npm installed on your PC first.
+
+
+
+### Setup
+
+Clone this repository to your desired folder:
+
+[https://github.com/seyioyemade/hello-world.git]
+
+
+
+### Install
+
+Install this project with:
+
+-Copy the URL [https://github.com/seyioyemade/hello-world.git].
+
+-In Visual Studio Code “Terminal” change the current working directory to the location where you want the cloned directory.
+
+-Type git clone in the terminal, paste the URL you copied earlier, and press “enter” to create your local clone.
+
+git clone [https://github.com/seyioyemade/hello-world.git]
+
```

Comment:
[ OPTIONAL ]
well done @seyioyemade, you implemented all this project requirements and applied all changes required by previous reviewers 💪, one small thing to mention is that the content of this section should move to setup and here all user have to do is run ' npm install '.